src/components/biggive-button/biggive-button.scss

```diff
@@ -5,7 +5,7 @@
 }
 
 .container {
-  @include standard-font();
+  font-family: inherit;
```

Comment:
is there a reason to think this will do anything on any browser? Hard to see how the font for a div could ever *not* be inherited. If not can we remove?

Reply:
MDN says the initial behaviour varies by browser if I read it right, so I thought that especially with shadow DOM involved it seemed safer to be explicit.

I'd prefer to test focused in on one component first I think.

Reply:
Right, I can see that the behaviour would vary in cases like here https://github.com/thebiggive/components/blob/db1cf4379f01a6fc5f7bfd5a4c6ab489c32e2bcf/src/components/biggive-nav-group/biggive-nav-group.scss#L7 where we set the font on a `ul`. In that case specifying `font-family: inherit` makes sense and is a typical css reset thing. But I imagine a div having any default other than inherit in any reasonable browser.

Fine to test on one component first of course.

Reply:
Yeah maybe so, the guidance was very general. Certainly with a single DOM involved I wouldn't have included this, I'm just not sure if different assumptions about the best amount of encapsulation could be made with a shadow one.

I suggest we do this first and try to confirm we've got the visual basics working just for PhilCo components and one or two browsers. Then we can try later to remove explicit `inherit`s along with any mixins that seem avoidable, and do a bigger test across more components, more browsers and both apps.